MIGRATE.md

```diff
@@ -56,12 +56,17 @@ server or note its path if the same server will be used.
     ```
     openssl rsa -in /root/smt-data/ssl/cacert.key -out /usr/share/rmt/ssl/rmt-ca.key
     ```
+    **Note:** if you use YaST2 RMT plugin with version `1.0.1` or newer just copy the key but not decrypt.
```

Comment:
> YaST2 RMT plugin with version `1.0.1`

What does this mean? Using the yast2-rmt plugin version 1.0.1, or using any version of the plugin with RMT 1.0.1?

Reply:
Why mention old versions at all? Let's just have the instructions for the most recent version.